docs/workflows/sign-images.rst

```diff
@@ -19,23 +19,23 @@ The example below demonstrates how a manifest signing service can be created usi
 
        {"signature_path": "signature"}
 
-   Below is an example of a signing script:
+   Below is a snippet of the signing script to be used for signing content:
 
    .. code-block:: bash
 
        #!/usr/bin/env bash
 
         MANIFEST_PATH1=$1
         FINGEPRINT="$PULP_SIGNING_KEY_FINGERPRINT"
-        REFERENCE="$IMAGE_REFERENCE"
-        SIG_PATH="$SIGNATURE_PATH"
+        IMAGE_REFERENCE="$REFERENCE"
+        SIGNATURE_PATH="$SIG_PATH"
```

Comment:
It might not be that straightforward for administrators to see where environment variables `$ REFERECE` and` $ SIG_PATH` come from. Are they set by them before running a task? (No). Maybe, it would be sufficient just to say that these variables are set inside a Pulp task.